lib/ansible/plugins/filter/combine.yml

```
@@ -0,0 +1,29 @@
+DOCUMENTATION:
+  name: combine
+  version_added: "2.0"
+  short_description: combine two dictionaries
+  description:
+    - Create a dictionary (hash/associative array) as a result of merging existing dictionaries
+  positional: _input, '*dicts'
```

Hmm, this syntax is currently breaking antsibull-docs (and thus causing the docs build errors), since it really doesn't expect this. I guess this either should be
```suggestion
  positional: _input, dicts
```
or below it should also be `*dicts` instead of `dicts`.

---

this is something that can be allowed multiple times, the * indicates multiple copies of the single option

---

The problem is that this syntax is completely new, and antsibull-docs doesn't know about it. So as long as you intend to use it and antsibull-docs is not updated (which requires containers to be rebuilt, so it won't happen that easily), either you have to adjust the syntax, or avoid it somehow.